http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -61,6 +63,62 @@ paths:
               schema:
                 $ref: "#/components/schemas/LeonardoVersion"
       security: []
+  /api/google/v1/runtime/{googleProject}/{name}:
```

Comment:
Once https://github.com/DataBiosphere/leonardo/pull/1321 is merged let's add the other routes